server/repository/migrations/sqlite/2023-04-21T10-00_name_table_add_columns_update_types/up.sql

```diff
@@ -0,0 +1,33 @@
+-- These changes were originally inserted into an earlier migration on develop,
+-- which means they won't run when updating to a "programs" branch if the
+-- earlier migrations had already run.
+
+-- 2021-08-15T10-00_create_name_table
+ALTER TABLE name ADD COLUMN gender_new TEXT;
+UPDATE name SET gender_new =  gender;
+ALTER TABLE name DROP COLUMN gender;
+ALTER TABLE name RENAME COLUMN gender_new TO gender;
+
+ALTER TABLE name ADD COLUMN is_deceased BOOLEAN;
+ALTER TABLE name ADD COLUMN national_health_number TEXT;
+
+-- 2022-03-15T10-00_report_table
+CREATE TABLE report_new
+(
+    id TEXT NOT NULL PRIMARY KEY,
+    name TEXT NOT NULL,
+    type TEXT CHECK (type IN ('OM_SUPPLY')) NOT NULL,
```

Comment:
The new fields are missing ('PATIENT' and  'DISPENSARY')... Or better just remove the check.

Reply:
Cool 90030202f